src/satellite_images_nso/_manipulation/nso_manipulator.py

```diff
@@ -33,7 +33,7 @@ def __make_the_crop(coordinates, raster_path, raster_path_cropped, plot):
     This crops the sattelite image with a chosen shape.
 
     TODO: Make this accept a object of geopandas or shapely and crs independant.
-    @param load_schape: path to a geojson shape file.
+    @param coordinates: Coordinates of the polygon to make the crop on.

```

Comment:
The 'plot' parameter is still missing in this docstring!

Reply:
should be fixed